src/api/common/errors.js

```diff
@@ -1,15 +1,34 @@
 /* eslint-disable valid-jsdoc */
 'use strict';
 
+const util = require('util');
+const _ = require('lodash');
+
 /**
  * Base class for all errors
  */
 function RippleError(message) {
   this.message = message;
 }
-RippleError.prototype = new Error();
+
+RippleError.prototype = Object.create(Error.prototype);
 RippleError.prototype.name = 'RippleError';
 
+
+RippleError.prototype.toString = function() {
+  let result = '[' + this.name + '(' + this.message;
+  if (this.data) {
+    result += ', ' + util.inspect(this.data);
+  }
+  result += ')]';
+  return result;
+};
+
+RippleError.prototype.inspect = function(depth) {
+  _.noop(depth);
```

Comment:
This is just for silencing eslint. I recommend defining a function called `unused` rather than `noop` to make it more clear.